studio/local_worker.py

```diff
@@ -77,7 +78,7 @@ def run(self, experiment):
                 python = which(python)
 
                 cmd = [python, experiment.filename] + experiment.args
-                cwd = experiment.artifacts['workspace']['local']
+                cwd = experiment.artifacts['workspace'].local_path
```

Comment:
Does naming this var explicitly named rather in a map hurt extensibility ? Not that we would extend it much of course.

Reply:
experiment.artifacts['workspace'] here is an instance of Artifact class, so 'local_path' is a property of this instance.